core/server/api/v2/utils/serializers/output/slack.js

```diff
@@ -0,0 +1,11 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:slack');
```

Comment:
seems like this whole serializer is also kind of redundant, would not add the code that is not needed :)  we don't expect anything to be returned for this endpoint anyways